crm_claim_code/models/crm_claim.py

```diff
@@ -29,3 +30,14 @@ def copy(self, default=None):
             default = {}
         default['code'] = self.env['ir.sequence'].get('crm.claim')
         return super(CrmClaim, self).copy(default)
+
+    @api.multi
+    @api.depends('code')
```

Comment:
No worries! I'm still finding cool new use cases with the new API even now. Makes some things so much easier to express.